plugins/c8y_remote_access_plugin/src/main.rs

```diff
@@ -64,6 +71,73 @@ fn remove_supported_operation(config_dir: &Path) -> miette::Result<()> {
         .with_context(|| format!("Removing supported operation at {}", path.display()))
 }
 
+static SUCCESS_MESSAGE: &str = "CONNECTED";
+
+#[derive(miette::Diagnostic, Debug, thiserror::Error)]
+#[error("Failed while {1}")]
+#[diagnostic(help(
+    "This should never happen. It's very likely a bug in the c8y remote access plugin."
+))]
+struct Unreachable<E: std::error::Error + 'static>(#[source] E, &'static str);
+
+async fn spawn_child(command: String) -> miette::Result<()> {
+    let mut command = tokio::process::Command::new("/usr/bin/c8y-remote-access-plugin")
```

Comment:
It would be better to use the $0 argument of the process, rather than assuming the plugin installed at a very specific location.